.drone.star

```diff
@@ -49,3 +51,78 @@ def tests(ctx, tests):
                 }
             ]
     return pipelines
+
+def release(ctx):
+    return [
+        {
+            "kind": "pipeline",
+            "name": "release",
+            "depends_on": [],
+            "steps": [
+                {
+                    "name": "zip",
+                    "image": OC_UBUNTU,
+                    "commands": [
+                        "apt-get update",
+                        "apt-get install -y zip",
+                        "mkdir ocis",
+                        "cp -r `ls | grep -v \"^ocis$\"` ocis/",
+                        "zip -r moodle-repository_ocis_%s.zip ocis" % (ctx.build.ref.replace("refs/tags/v", "")),
+                        "ls -lh"
+                    ],
+                },
+                {
+                    "name": "release to GitHub",
+                    "image": PLUGINS_GITHUB_RELEASE,
+                    "settings": {
+                        "api_key": {
+                            "from_secret": "github_token",
+                        },
+                        "files": [
+                            "moodle-repository_ocis_%s.zip" % (ctx.build.ref.replace("refs/tags/v", "")),
+                            ],
+                        "title": ctx.build.ref.replace("refs/tags/v", ""),
+                        "note": "CHANGES.md",
+                        "overwrite": True,
+                        "prerelease": len(ctx.build.ref.split("-")) > 1,
+                    },
+                },
+                {
+                    "name": "release to moodle",
+                    "image": OC_UBUNTU,
+                    "environment": {
+                        "ZIPURL": "https://github.com/owncloud/moodle-repository_ocis/releases/download/%s/moodle-repository_ocis_%s.zip" % (ctx.build.ref.replace("refs/tags/", ""), ctx.build.ref.replace("refs/tags/v", "")),
+                        "TAGNAME": ctx.build.ref.replace("refs/tags/v", ""),
+                        "MOODLETOKEN": {
+                            "from_secret": "moodle_token",
+                        },
+                    },
+                    "commands": [
+                        "RESPONSE=$(curl https://moodle.org/webservice/rest/server.php \
+                            --data-urlencode \"wstoken=$MOODLETOKEN\" \
+                            --data-urlencode \"wsfunction=local_plugins_add_version\" \
+                            --data-urlencode \"moodlewsrestformat=json\" \
+                            --data-urlencode \"frankenstyle=repository_ocis\" \
+                            --data-urlencode \"zipurl=$ZIPURL\" \
+                            --data-urlencode \"vcssystem=git\" \
+                            --data-urlencode \"vcsrepositoryurl=https://github.com/owncloud/moodle-repository_ocis/\" \
+                            --data-urlencode \"vcstag=$TAGNAME\" \
+                            --data-urlencode \"changelogurl=https://github.com/owncloud/moodle-repository_ocis/releases/tag/$TAGNAME\" \
```

Comment:
```suggestion
                            --data-urlencode \"changelogurl=https://github.com/owncloud/moodle-repository_ocis/releases/tag/v$TAGNAME\" \
```

IMO the GitHub URL needs to have the full release version with the "v" included.